substrate/frame/support/src/lib.rs

```diff
@@ -2181,9 +2208,72 @@ pub mod pallet_macros {
 	///   [`crate::dispatch::DispatchResultWithPostInfo`]).
 	///
 	/// **WARNING**: modifying dispatchables, changing their order (i.e. using [`call_index`]),
-	/// removing some, etc., must be done with care. This will change the encoding of the , and
-	/// the call can be stored on-chain (e.g. in `pallet-scheduler`). Thus, migration might be
-	/// needed. This is why the use of `call_index` is mandatory by default in FRAME.
+	/// removing some, etc., must be done with care. This will change the encoding of the call,
+	/// and the call can be stored on-chain (e.g. in `pallet-scheduler`). Thus, migration
+	/// might be needed. This is why the use of `call_index` is mandatory by default in FRAME.
+	///
+	/// ## Weight info
+	///
+	/// ```
```

Comment:
```suggestion
	/// Each call needs to define a weight.
	/// * The weight can be defined explicitly using the attribute `#[pallet::weight($expr)]` (Note that argument of the call are available inside the
	/// expression).
	/// * Or it can be defined implicitly, the weight info for the calls needs to be specified in the call attribute: `#[pallet::call(weight = $WeightInfo)]`, then each call that doesn't have explicit weight will use `$WeightInfo::$call_name` as the weight.
	/// * Or it can be simply ignored when the pallet is in `dev_mode`.
	///
	/// ```
```